test/CheckUnusedTest.hs

```diff
@@ -0,0 +1,59 @@
+module CheckUnusedTest where
+
+import           Lib                                      ( Expr
```

Comment:
Looks like something happened to the whitespace distribution here. Auto-formatting?